src/js/facility-locator/components/ServicesAtFacility.jsx

```diff
@@ -131,15 +129,16 @@ class ServicesAtFacility extends Component {
     // TODO: clean up once we have real data
     return (
       <div>
-        {facility.type === 'va_health_facility' ? this.renderHealthServices() : null}
+        <p style={{ margin: '0 0 0.5em' }}>Services current as of <strong>{moment().format('MMMM D, YYYY')}</strong></p>
```

Comment:
the goal on the date is to have it so when services get updates and refreshed on details page it is reflected in a "current as of date".  

Dat specifically asked to make sure we have that.